### ISSUE TITLE
Cannot find published package from sonatype

### ISSUE THREAD

**Issue by chiplet** [chipsalliance/cde#28]
Looking at the `build.sc` and github actions configuration, it seems like this library should be deployed to sonatype automatically. The latest CI workflow reports as having published the package successfully. However, I am unable to find the published packages from anywhere. So far I've tried searching from:
- https://s01.oss.sonatype.org,
- https://oss.sonatype.org, and
- https://search.maven.org/ 

There is a match for `me.sequencer.cde` on maven central, but that seems to be from 2021. Where can I find the latest package published as `org.chipsalliance.cde`?

**Comment by sequencer** [comment]
Yes, this is intended, I'm trying fixing the publish issue recently.
